install.sh

```diff
@@ -26,36 +26,43 @@ echo
 echo "SA201H mapping is for remapping button closer to spike as middle click and second button as right click"
 echo
 PS3='Please enter your choice '
-options=($(ls stylus_layouts) "Quit")
+options=($(ls src/layouts) "Quit")
 select selected_opt in "${options[@]}"
 do
     if [ "$selected_opt" = "Quit" ]
     then
         exit 0
     fi
 
-    for option in $(ls stylus_layouts);
+    for option in $(ls src/layouts);
     do
         if [ "$option" = "$selected_opt" ] ; then
-            model=${selected_opt::-3}
+            layout=${selected_opt%.py}
             break
         fi
     done
 
-    if [ -z "$model" ] ; then
+    if [ -z "$layout" ] ; then
         echo "invalid option $REPLY"
     else
         break
     fi
 done
 
-echo "Add asus stylus service in /etc/systemd/system/"
-cat asus_stylus.service | LAYOUT=$layout envsubst '$LAYOUT' > /etc/systemd/system/asus_stylus.service
+echo "Add asus stylus service in /usr/lib/systemd/system/"
+install -v -Dm644 -t /usr/lib/systemd/system src/asus-stylus.service
```

Comment:
Package is atm designed to be installed as system-wide (not user-only), why then put the package under `/usr/lib..`?